doc/dev/placement-group.rst

```diff
@@ -45,12 +45,11 @@ is the primary and the rest are replicas.
 Many PGs can map to one OSD.
 
 A PG represents nothing but a grouping of objects; you configure the
-number of PGs you want (see
-http://ceph.com/wiki/Changing_the_number_of_PGs ), number of
-OSDs * 100 is a good starting point, and all of your stored objects
-are pseudo-randomly evenly distributed to the PGs. So a PG explicitly
-does NOT represent a fixed amount of storage; it represents 1/pg_num
-'th of the storage you happen to have on your OSDs.
+number of PGs you want, number of OSDs * 100 is a good starting point
```

Comment:
i searched it, but i can't find it in our wiki. I think it's ok to remove this link